src/Components/itemDetail.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import '../App.css';
+import { useParams } from 'react-router-dom';
```

Comment:
The useParams hook is interesting, I'm glad to now know how to link to specific items (ie blog posts, content, etc.).  There was a previous project I was working on that could have used this exact hook.